tests/test_container.py

```diff
@@ -1,6 +1,7 @@
 import struct
```

Comment:
For the Union types in `test_open_inputs()`
```suggestion
from __future__ import annotations

import struct
```